dev-ruby/net-ssh/net-ssh-6.2.0-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
This can be dropped, I think, because vagrant now requires >=net-ssh-6.1 again, so there is no need for a release candidate. In general I'm hesitant to add release candidates because it increases the maintenance burden, and for security sensitive packages like this I try to avoid it altogether.

Reply:
Yes you are right,
I dropped this additional ebuild and  added my changes to enable tests on 6.1.0-r1